parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -666,17 +661,15 @@ impl snowbridge_ethereum_beacon_client::Config for Runtime {
 	type WeightInfo = weights::snowbridge_ethereum_beacon_client::WeightInfo<Runtime>;
 }
 
-parameter_types! {
-	pub const GovernanceProxyContract: snowbridge_core::ContractId = snowbridge_core::ContractId::new(hex_literal::hex!("44bef07c29162ad04096f5cbe78ca2df62dffe97cea85825f08d13319e13f34a"));
-}
-
 impl snowbridge_control::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type OwnParaId = ParachainInfo;
-	type OutboundQueue = EthereumOutboundQueue;
-	type GovernanceProxyContract = GovernanceProxyContract;
 	type MessageHasher = BlakeTwo256;
+	type OutboundQueue = EthereumOutboundQueue;
+	type OwnParaId = ParachainInfo;
 	type WeightInfo = ();
+	type MaxUpgradeDataSize = ();
+	type CreateAgentOrigin = EnsureXcmOrigin<RuntimeOrigin, ()>;
```

Comment:
Not sure what the second parameter should be and if this is correct.

Reply:
@vgeddes @alistair-singh is this OK?

Reply:
@alistair-singh ^

Reply:
Nevermind, I got my answer from changing the merge base branch to `proxy-contracts`.